subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/IsolatedAntBuilderMemoryLeakIntegrationTest.groovy

```diff
@@ -43,7 +44,7 @@ class IsolatedAntBuilderMemoryLeakIntegrationTest extends AbstractIntegrationSpe
         """
     }
 
-    private void withCodenarc(String groovyVersion, TestFile root = testDirectory) {
+    private void withCodenarc(String groovyVersion, String codenarcVersion = '0.24.1', TestFile root = testDirectory) {
```

Comment:
This default version is quite old, should we consider testing with a newer version like we are with groovy 4? Perhaps we could consider updating this test to a `MultiVersion` test, and test the versions in `CodeNarcCoverage`.